cmp/tab/TabPane.js

```diff
@@ -40,6 +40,7 @@ export class TabPane extends Component {
         return frame({
             display: isActive ? 'flex' : 'none',
             margin: 4,
+            cls: 'xh-tab-component',
```

Comment:
Why not `xh-tab-pane`? (I admit it is a bit of a funny name in general, but might as well keep this new CSS class consistent with the component.)